modules/ossm-threescale-integrate-1x.adoc

```diff
@@ -21,15 +20,17 @@ You can use these examples to configure requests to your services using the 3sca
 To configure the 3scale Istio Adapter, refer to {ProductName} custom resources for instructions on adding adapter parameters to the custom resource file.
 ====
 
-
 [NOTE]
 ====
-Pay particular attention to the `kind: handler` resource. You must update this with your 3scale credentials and the service ID of the API you want to manage.
+Pay particular attention to the `kind: handler` resource. You must update this with your 3scale account credentials. You can optionally add a `service_id` to a handler, but this is kept for backwards compatibility only, since it would render the handler only useful for one service in your 3scale account. Enabling 3scale for other services will require for you to create more handlers which only differ in their `service_id`s.
```

Comment:
I'd rewrite the last sentence to make it clear that it will only happen if the user goes the route of specifying the service_id in the handler: `If you add service_id to a handler, enabling 3scale for other services will require for you to create more handlers which only differ in their service_id's.`

Edit: note this also applies as well to the same text in modules/ossm-threescale-integrate.adoc.